src/components/UserManagement/SetUpFinalDayButton.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Button } from 'reactstrap';
+import { SET_FINAL_DAY, CANCEL } from '../../languages/en/ui';
+import SetUpFinalDayPopUp from './SetUpFinalDayPopUp';
+import { updateUserFinalDayStatus } from 'actions/userManagement';
+import axios from 'axios';
+import { ENDPOINTS } from 'utils/URL';
+import { toast } from 'react-toastify';
+/**
+ * @param {*} props
+ * @param {Boolean} props.isBigBtn
+ * @param {*} props.userProfile.isSet
+ * @returns
+ */
+const SetUpFinalDayButton = (props) => {
+  const[isSet, setIsSet] = useState(false);  
+  const[finalDayDateOpen, setFinalDayDateOpen] = useState(false);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (props.userProfile?.endDate !== undefined)
+      setIsSet(true);
+  }, [setIsSet]);
+  
+  const onFinalDayClick = (user, status) => {
+    if(isSet){
+      updateUserFinalDayStatus(props.userProfile,"Active",undefined)(dispatch)
+      setIsSet(!isSet);
+      toast.success('Your Final Day has been set.');
```

Comment:
Consider using `This user's final day has been set`, similar apply to line 46

Reply:
thx, already change form 'Your Final Day has been set.' to 'This user's final day has been set', also in line 46